make/photon/clair-adapter/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM photon:2.0
+
+RUN tdnf install -y sudo >>/dev/null\
+    && tdnf clean all \
+    && mkdir /clair-adapter/ \
+    && groupadd -r -g 10000 clair-adapter \
+    && useradd --no-log-init -m -r -g 10000 -u 10000 clair-adapter
+
+COPY ./make/photon/clair-adapter/binary/harbor-scanner-clair /clair-adapter/clair-adapter
+
+RUN chown -R 10000:10000 /clair-adapter \
```

Comment:
Should we run this container as non-root user?

Reply:
Yes